tox.ini

```diff
@@ -16,6 +16,9 @@ norecursedirs = .* docs requirements
 [testenv]
 deps =
     django111: Django>=1.11,<2.0
+    django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
+    django22: Django>=2.2,<2.3
     -r{toxinidir}/requirements/test.txt
 commands =
     py.test {posargs}
```

Comment:
I think that in order to display deprecation warnings, we need to update this to something like `python -Wd -m pytest {posargs}` (although there might not be any warnings triggered by the code in this repo, it's pretty small).